README.md

```diff
@@ -427,6 +427,7 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
 | NPM                | ![NPM](https://img.shields.io/badge/NPM-%23CB3837.svg?style=for-the-badge&logo=npm&logoColor=white)                                             | `![NPM](https://img.shields.io/badge/NPM-%23CB3837.svg?style=for-the-badge&logo=npm&logoColor=white)`                                             |
 | NestJS             | ![NestJS](https://img.shields.io/badge/nestjs-%23E0234E.svg?style=for-the-badge&logo=nestjs&logoColor=white)                                    | `![NestJS](https://img.shields.io/badge/nestjs-%23E0234E.svg?style=for-the-badge&logo=nestjs&logoColor=white)`                                    |
 | Next JS            | ![Next JS](https://img.shields.io/badge/Next-black?style=for-the-badge&logo=next.js&logoColor=white)                                            | `![Next JS](https://img.shields.io/badge/Next-black?style=for-the-badge&logo=next.js&logoColor=white)`                                            |
+| Nextra             | ![Nextra](https://img.shields.io/badge/nextra-black.svg?style=for-the-badge&logo=nextra&logoColor=white)                                        | `![Nextra](https://img.shields.io/badge/nextra-black.svg?style=for-the-badge&logo=nextra&logoColor=white)`                                        |
```

Comment:
Icons are already there and tell what is missing

Reply:
What is the meaning of badge

Reply:
![image](https://github.com/Ileriayo/markdown-badges/assets/66284362/3ac0e3c8-a623-4436-b60a-c2a8fd21d83c)

A snapshot from the Nextra webpage. Sadly, the Nextra Icon is missing from the badge. 
A badge looks like this![Next JS](https://img.shields.io/badge/Next-black?style=for-the-badge&logo=next.js&logoColor=white)

Reply:
Somehow, the `https://img.shields.io/badge` API does not return the nextra SVG icon and API like `https://unpkg.com/simple-icons@v9/icons/nextra.svg` is work fine.